### PR TITLE
fix wagtail media upload error

### PR DESCRIPTION
**Pull request by umarmughal824**
#### Pre-Flight checklist

- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
NA

#### What's this PR do?
fix 502 error while uploading images on wagtail

#### How should this be manually tested?
I have tested it by adding AWS and S3 keys locally

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
+# This file is autogenerated by pip-compile with python 3.9
```
Comment (pdpinch) [inline requirements.txt:2]: Just checking -- this app _is_ on python 3.9, right?

Comment (umarmughal824) [reply requirements.txt:2]: yeah, it's running on python 3.9

**Review by pdpinch — APPROVED** [review]
Seems necessary, and what harm can it do?

File: requirements.txt (hunk 1 above)
Comment (umarmughal824) [reply requirements.txt:2]: https://github.com/mitodl/mitxonline/blob/main/Dockerfile#L1

**Comment by umarmughal824** [comment]
> Seems necessary, and what harm can it do?

no harm at all

**Pull request merged**